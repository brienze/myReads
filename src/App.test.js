
import React from 'react';
import { shallow } from 'enzyme';
import MyReadsApp from './App';


it('should render without any errors', () => {
  expect(shallow(<MyReadsApp/>)).toMatchSnapshot();
});
