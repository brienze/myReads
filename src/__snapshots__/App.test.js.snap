// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without any errors 1`] = `
<div
  className="app"
>
  <Route
    exact={true}
    path="/"
    render={[Function]}
  />
  <Route
    path="/search"
    render={[Function]}
  />
</div>
`;
